gnome-initial-setup/pages/live-chooser/gis-live-chooser-page.c

```diff
@@ -51,6 +51,19 @@ typedef struct _GisLiveChooserPagePrivate GisLiveChooserPagePrivate;
 G_DEFINE_TYPE_WITH_PRIVATE (GisLiveChooserPage, gis_live_chooser_page, GIS_TYPE_PAGE);
 
 static void
+disable_chrome_auto_download (GisLiveChooserPage *page)
+{
+  GError *error = NULL;
+
+  if (!gis_pkexec (DATADIR "/eos-google-chrome-helper/eos-google-chrome-system-helper.py",
+                   NULL, &error))
+    {
+      g_warning ("Failed to disable Chrome auto-download: %s\n", error->message);
```

Comment:
@mcatanzaro I add a commit at the beginning of the rebase to be sure there were not conflicts. title: `remove unnecessary '\n' termination character`. This should be good to submit upstream.